Tools-Override/tests.targets

```diff
@@ -211,8 +211,54 @@
       <TestCommandLines Include="cd native"/>
     </ItemGroup>
 
+    <!-- For UAP F5 scenario, generate the test commands -->
+    <Error Condition="'$(BuildingUAPVertical)' == 'true' and '$(UAPRunnerFolder)'==''" Text="Please specify UAPRunnerFolder" />
+    <Error Condition="'$(BuildingUAPVertical)' == 'true' and '$(UAPLauncherFolder)'==''" Text="Please specify UAPLauncherFolder" />
+
+    <PropertyGroup Condition="'$(BuildingUAPVertical)' == 'true'">
+      <_Runner_Path>$(UAPRunnerFolder)\</_Runner_Path>
+      <_Launcher_Path>$(UAPLauncherFolder)\WindowsStoreAppLauncher.exe</_Launcher_Path>
+      <_Runtime_Path>$(RuntimePath)</_Runtime_Path>
+      <_Results_Path>$([System.Environment]::GetFolderPath(SpecialFolder.MyDocuments))\TestResults\</_Results_Path>
+    </PropertyGroup>
+  
+    <!-- The paths are different on the helix machine than locally -->
+    <PropertyGroup Condition="'$(BuildingUAPVertical)' == 'true' and '$(ArchiveTests)' == 'true'">
+      <_Runner_Path>%RUNTIME_PATH%\Runner\</_Runner_Path>
+      <_Launcher_Path>%RUNTIME_PATH%\Launcher\WindowsStoreAppLauncher.exe</_Launcher_Path>
+      <_Runtime_Path>%RUNTIME_PATH%\uap-f5-layout\</_Runtime_Path>
+      <_Results_Path>%USERPROFILE%\Documents\TestResults\</_Results_Path>
+    </PropertyGroup>
+
+    <PropertyGroup Condition="'$(BuildingUAPVertical)' == 'true'">
+      <!-- Make sure the path only has one type of slashes -->
+      <_Runtime_Path>$(_Runtime_Path.Replace('/', '\'))</_Runtime_Path>
+
+      <!-- The test execution command is different for UAP. -->
+      <TestCommandLine>$(_Launcher_Path) -test appxmanifest.xml $(XunitArguments) $(XunitTraitOptions)</TestCommandLine>
+    </PropertyGroup>
+
+    <ItemGroup Condition="'$(BuildingUAPVertical)' == 'true'" >
```

Comment:
I know I told you to use BuildingUAPVertical but I do worry a little bit about that causing issues later because there is a decent chance that we will start using the BuildAllConfigurations to build the test projects as well and in that case I think all hell will break loose in these test targets.